docs/modules/ROOT/examples/README.adoc

```diff
@@ -0,0 +1 @@
+../../../../README.adoc
```

Comment:
We are using symlinks because Antora cannot reference files outside of the documentation component (i.e., outside of the docs directory)